Library/Formula/mitie.rb

```diff
@@ -14,27 +14,37 @@ class Mitie < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
+  option "without-models", "Don't download the v0.2 models (~415MB)"
+
   resource "models-english" do
-    url "http://sourceforge.net/projects/mitie/files/binaries/MITIE-models-v0.2.tar.bz2"
-    sha1 "a193e0a133926b5355b4e840ddd7f5112d785999"
+    url "https://downloads.sourceforge.net/project/mitie/binaries/MITIE-models-v0.2.tar.bz2"
+    sha256 "dc073eaef980e65d68d18c7193d94b9b727beb254a0c2978f39918f158d91b31"
   end
 
   def install
-    (share+"MITIE-models").install resource("models-english")
+    if build.with? "models"
+      (share/"MITIE-models").install resource("models-english")
+    end
+
     inreplace "mitielib/makefile" do |s|
-      s.gsub! /libmitie.so/, "libmitie.dylib"
+      s.gsub!(/libmitie.so/, "libmitie.dylib")
     end
     system "make", "mitielib"
     system "make"
+
+    include.install Dir["mitielib/include/*"]
     lib.install "mitielib/libmitie.dylib", "mitielib/libmitie.a"
-    (lib+'python2.7/site-packages').install "mitielib/mitie.py"
-    (share+"mitie").install "examples", "sample_text.txt",
+    (lib/"python2.7/site-packages").install "mitielib/mitie.py"
+    (share/"mitie").install "examples", "sample_text.txt",
       "sample_text.reference-output", "sample_text.reference-output-relations"
     bin.install "ner_example", "ner_stream", "relation_extraction_example"
   end
 
   test do
-    system "#{bin}/ner_example", "#{share}/MITIE-models/english/ner_model.dat", "#{share}/mitie/sample_text.txt"
+    system ENV.cc, "-I#{include}", "-L#{lib}", "-lmitie",
+           share/"mitie/examples/C/ner/ner_example.c",
+           "-o", testpath/"ner_example"
+    system "./ner_example", share/"MITIE-models/english/ner_model.dat",
+           share/"mitie/sample_text.txt"
```

Comment:
I can do that, but I've been told previously not to worry about the test passing with non-default options.